src/subscription/subscription.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { DbService } from 'src/db/db.service';
+
+interface Subscription {
```

Comment:
Should be able to get this from Prisma but this is good for now.

Reply:
Remove the custom type and used the prisma generated one.